### PR TITLE
Implement Display models for QualifiedConcept

### PR DESCRIPTION
**Pull request by alexwlchan**
A stepping stone to #1682. Builds on #1715.

### CODE REVIEW

File: common/src/test/scala/uk/ac/wellcome/models/ConceptTest.scala [hunk 1]
```diff
@@ -0,0 +1,69 @@
+package uk.ac.wellcome.models
+
+import org.scalatest.{FunSpec, Matchers}
+import org.scalatest.prop.TableDrivenPropertyChecks._
+import uk.ac.wellcome.test.utils.JsonTestUtil
+import uk.ac.wellcome.utils.JsonUtil._
+
+class ConceptTest extends FunSpec with Matchers with JsonTestUtil {
+
+  val testCases = Table(
```
Comment (alicefuzier) [inline common/src/test/scala/uk/ac/wellcome/models/ConceptTest.scala]: Same as in the other PR. I don't really like this usage of property testing to shrink test cases. If all of the elements in this table are relevant tests cases then that should be made explicit by separating them in different tests

**Comment by alicefuzier** [comment]
Looks good to me except the test thing and I'd rather get to a common agreement on how we write these tests before adopting this pattern everywhere

File: common/src/test/scala/uk/ac/wellcome/models/ConceptTest.scala (hunk 1 above)
Comment (rgladwell) [reply common/src/test/scala/uk/ac/wellcome/models/ConceptTest.scala]: There are a few tests similar to this and I think I might be missing something: they seem to verify that the case classes serialize and deserialize. Is this testing/verifying anything else?

Comment (alicefuzier) [reply common/src/test/scala/uk/ac/wellcome/models/ConceptTest.scala]: No really. to be honest I wonder whether they even need to exist. I don't think all of the model classes have them (I don't think I added them for Agent). It's also tested again in `IdentifiedWorkTest` and `UnidentifiedWorkTest`

**Pull request closed without merging**